app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,8 @@
+class RecipesController < ApplicationController
+    def index
```

Comment:
Careful about adding an action to a controller that is unimplemented; can be a security concern down the line. Not a big deal now, just something to be aware of.